restapi/src/test/java/io/stargate/web/resources/WhereParserTest.java

```diff
@@ -119,6 +119,26 @@ public void testParseValueEmpty() {
             "Value entry for field name, operation $gt was expecting a value, but found null.");
   }
 
+  // 04-Jan-2022, tatu: Verifies existing behavior as used by Stargate REST 1.0,
```

Comment:
Test is correct, but might look wrong if someone doesn't realize  that the behavior of $exists is expected to be equivalent to $EQ. It might be helpful to add a note here.

Reply:
Good point; will add bit more info.